### PR TITLE
audio: base_fw: add platform layer to fw_config data

### PR DESCRIPTION
**Pull request by kv2019i**
Build on foundation set in #9012

### CODE REVIEW

File: src/platform/intel/ace/lib/base_fw_platform.c [hunk 1]
```diff
@@ -9,6 +9,24 @@
 #include <sof/lib/dai.h>
 #include <ipc4/base_fw.h>
 
+int platform_basefw_fw_config(uint32_t *data_offset, char *data)
+{
+	struct sof_tlv *tuple = (struct sof_tlv *)data;
+
+	tlv_value_uint32_set(tuple, IPC4_SLOW_CLOCK_FREQ_HZ_FW_CFG, IPC4_ALH_CAVS_1_8);
```
Comment (ujfalusi) [inline src/platform/intel/ace/lib/base_fw_platform.c:16]: This cannot be right, right?
see the original code:
```c
	tuple = tlv_next(tuple);
	tlv_value_uint32_set(tuple,
			     IPC4_SLOW_CLOCK_FREQ_HZ_FW_CFG,
			     clock_get_freq(CPU_LOWEST_FREQ_IDX));

	tuple = tlv_next(tuple);
	tlv_value_uint32_set(tuple, IPC4_SLOW_CLOCK_FREQ_HZ_FW_CFG, IPC4_ALH_CAVS_1_8);
```
The first set of `IPC4_SLOW_CLOCK_FREQ_HZ_FW_CFG` looks valid, the second (and the one which got moved) is not.

Comment (kv2019i) [reply src/platform/intel/ace/lib/base_fw_platform.c:16]: @ujfalusi I'm not touching the values here, just moving them around as-is. That certainly looks wrong, but haven't dug into details yet. Linux driver is not using this, so hard to say much about this.

**Review by lgirdwood — COMMENTED** [review]
Lets do the code move 1st, we can then followup if we thing data is wrong (as its passing CI).
@softwarecki pls review.

**Comment by kv2019i** [comment]
There's more incoming, now still as draft in #9060

**Pull request closed without merging**